content/en/docs/collector/custom-collector.md

```diff
@@ -227,7 +244,115 @@ You can now use the generated code to bootstrap your component development
 projects and easily build and distribute your own collector distribution with
 your components.
 
-Further reading:
+## Step 3b - Containerizing your Collector's distribution
+
+{{% alert color="primary" title="NOTE" %}}
+
+This step will build your collector distribution in-place inside a `Dockerfile`.
```

Comment:
What does in place mean in this context?

Reply:
@jaronoff97 I mean that it's doing it inside the Dockerfile, rather than outside of it. But if that's confusing, I can totally remove that wording.